### PR TITLE
Added Distraction Free Plugin

### PR DESCRIPTION
**Pull request by wczekalski**
😉

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -2,6 +2,12 @@
   "packages": {
     "plugins": [
       {
+      "name": "Distraction free mode - ZEN",
```
Comment (jurre) [inline packages.json]: > The url should be a repository containing the Xcode project, and the name should be the same as the .xcodeproj file.

**Comment by wczekalski** [comment]
@jurre it should be ok now. I adjusted the project

**Comment by jurre** [comment]
It looks like the name still does not match the `.xcodeproj` file, or am I missing something?

**Comment by wczekalski** [comment]
duh, I forgot to push. Sorry!

**Comment by jurre** [comment]
:tada: thanks

**Pull request merged**